File: gdlocation/src/main/java/com/xhl/gdlocation/service/GdLocationInfo.java
package com.xhl.gdlocation.service;

/**
 * @author xhl
 * @desc 2019/3/13 10:19
 */
public class GdLocationInfo {
    private double longitude;
    private double latitude;
    private String disCode;
    private String province;
    private String city;
    private String district;

    public String getDisCode() {
        return disCode;
    }

    public void setDisCode(String disCode) {
        this.disCode = disCode;
    }

    public String getProvince() {
        return province;
    }

    public void setProvince(String province) {
        this.province = province;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getDistrict() {
        return district;
    }

    public void setDistrict(String district) {
        this.district = district;
    }

    private String pcd;//省市区

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public String getPcd() {
        return pcd;
    }

    public void setPcd(String pcd) {
        this.pcd = pcd;
    }


}

File: gdlocation/src/main/java/com/xhl/gdlocation/GdLocationInfo.kt
package com.xhl.gdlocation

/**
 * @author xhl
 * 封装自己需要的内容
 * @desc 2019/3/13 10:19
 */
class GdLocationInfo {
    var longitude: Double = 0.0
    var latitude: Double = 0.0
    var province:String=""
    var pcd: String = ""//省市区  高德地图不返回null 而是返回"" 省的到时候StringBuilder时将null-->"null"(百度返回的是null)
}
